src/System.Data.SqlClient/dir.props

```diff
@@ -3,7 +3,7 @@
   <Import Project="..\dir.props" />
   <PropertyGroup>
     <PackageVersion>4.4.2</PackageVersion>
```

Comment:
Package version needs to be bumped to 4.4.3

Reply:
Fixed to `4.4.3`